WapoJS/src/js_eval.rs

```diff
@@ -180,6 +180,10 @@ async fn run_with_service(
     }
     #[cfg(feature = "native")]
     {
+        let default_fn = js_ctx.get_global_object().get_property("module")?.get_property("exports").unwrap_or_default();
```

Comment:
1. Looks like this should be a common feature cross native and wasm? So, better move out from the "native" feature gate.
2. Keep in mind the question mark in `get_property("module")?` will return current fn on Error.